im3components/statemod_to_parquet/statemod_data_extraction.py

```diff
@@ -106,9 +106,10 @@ def __init__(
         # how many characters allotted to each field, not including whitespace between fields
         # note that in some files, several asterisks spilled into other fields, ruining ability to split by whitespace
         self.expected_column_sizes = np.asarray([
-            11, 12, 4, 4, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 12, 11
+            11, 13, 5, 5, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 13, 12
         ])
-        self.expected_line_size = self.expected_column_sizes.sum() + len(self.expected_column_sizes)
+        # the line separator counts as an extra character, hence the +1
```

Comment:
@thurber is there always a single separator or should this be stripped?

Reply:
it's a good question -- i'm fairly certain there will always be a single line separator. Stripping it might be problematic since there could be other white space at the beginning and end of the line that is important for counting the spacing between columns